specs/pvs/corla.pvs

```diff
@@ -486,11 +527,15 @@ END ballot_manifests
 % @relation well_formed_vote inherit vote
```

Comment:
Thus, @mkiniry is going over the glossary (and eventually the C.R.S. and law) to double-check that each instance of such a binary classifier is encoded in our specification in the same fashion.  We may choose to be more explicit/plain-spoken in our syntax, of course, and can avoid the `pred` notation.  But since this is the standard way of writing things in HOL in PVS, I'm doing it that way by default.